packages/las-sdk-core/src/client.spec.ts

```diff
@@ -629,6 +651,97 @@ describe('Batches', () => {
 });
 
 describe('Models', () => {
+  describe('createModel', () => {
+    test.each<[FieldConfig, number, number, CreateModelOptions | undefined]>([
+      [
+        {
+          total_amount: { type: 'amount', maxLength: 20, description: 'Total Amount' } as Field,
+          purchase_date: { type: 'date', maxLength: 10, description: 'Purchase Date' } as Field,
+          supplier_id: { type: 'alphanum', maxLength: 25, description: 'Supplier ID' } as Field,
+        } as FieldConfig,
+        100,
+        100,
+        {
+          preprocessConfig: { autoRotate: true, imageQuality: 'HIGH', maxPages: 3 } as PreprocessConfig,
+          name: 'My model name',
+          description: 'My model description',
+        } as CreateModelOptions
+      ],
+      [
+        {
+          total_amount: { type: 'amount', maxLength: 20, description: 'Total Amount' } as Field,
+          purchase_date: { type: 'date', maxLength: 10, description: 'Purchase Date' } as Field,
+          supplier_id: { type: 'alphanum', maxLength: 25, description: 'Supplier ID' } as Field,
+        } as FieldConfig,
```

Comment:
Are these type castings necessary? (`as Field` etc). I think it should work since you have the types in the generic parameter for the `test.each` function?